integration_test/third_party_apps_data/applications/mysql/metric_name.txt

```diff
@@ -1 +1 @@
-agent.googleapis.com/mysql/command_count
+agent.googleapis.com/cpu/utilization
```

Comment:
[Optional] Is the `cpu/utilization` metrics here just as a place holder before we add metrics support? We can skip the metrics checking by leaving this file empty.

Reply:
Yup, I was intending to come back and fix that once I had validated the empty file in Apache PR, and forgot, thanks!

Reply:
Thanks for fixing it!